cli/util/util.go

```diff
@@ -243,6 +246,28 @@ func GetLastNLines(filepath string, linesN int) ([]string, error) {
 	return lines, nil
 }
 
+func GetTarantoolVersion(cli *cmdcontext.CliCtx) (string, error) {
```

Comment:
It's a public function. Add description.

Reply:
Added.